components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -145,10 +148,10 @@ void BraveReferralsService::Start() {
 
   // On first run, read the promo code from user-data-dir and
   // initialize the referral.
-  bool checked_for_promo_code_file =
-      pref_service_->GetBoolean(kReferralCheckedForPromoCodeFile);
```

Comment:
If we're no longer using `kReferralCheckedForPromoCodeFile`, we should remove it.

Reply:
If the referral server is down for multiple days, should it affect our calculation regarding the number of days we wait to check for finalization (30 days) or the number of days after which we discard the referral code (90 days)?

Reply:
Other files still use `kReferralCheckedForPromoCodeFile`. Maybe they should be using the new preference instead?

Reply:
I would think so. I think the new preference you added supersedes this one now.